app/code/Magento/Cms/Controller/Adminhtml/Page/MassStatus.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Controller\Adminhtml\Page;
+
+use Magento\Backend\App\Action\Context;
+use Magento\Cms\Model\ResourceModel\Page\CollectionFactory;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Ui\Component\MassAction\Filter;
+
+/**
+ * Class MassStatus
+ * @package Magento\Cms\Controller\Adminhtml\Page
+ */
+class MassStatus extends \Magento\Backend\App\Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'Magento_Cms::save';
+
+    /**
+     * @var Filter
+     */
+    protected $filter;
```

Comment:
Since this is a new class we can update these from protected.

Reply:
Sure, but it only makes sense if we are going to maintain new controller class, let's discuss about it in the main thread